modules/database/src/ioc/db/Makefile

```diff
@@ -13,6 +13,7 @@ SRC_DIRS += $(IOCDIR)/db
 
 INC += callback.h
 INC += dbAccess.h
+INC += dbAccessNoThrow.h
```

Comment:
This header should not be installed (aka. become a supported public API).

Reply:
My bad. I will remove this.